egs/chime4/s5_1ch/local/compute_stoi_estoi_sdr.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Copyright 2017 Johns Hopkins University (Author: Aswin Shanmugam Subramanian)
+# Apache 2.0
+
+. ./cmd.sh
+. ./path.sh
+set -e
+set -u
+set -o pipefail
+
+njobs=10
+cmd=run.pl
+
+. utils/parse_options.sh || exit 1;
+
+if [ $# != 3 ]; then
+   echo "Wrong #arguments ($#, expected 3)"
+   echo "Usage: local/compute_stoi_estoi_sdr.sh [options] <enhancement-method> <enhancement-directory> <chime-rir-directory>"
```

Comment:
Give an xample of usage, and please explain a bit more.  E.g. what are the options for <enhancement-method>?

Reply:
Added documentation and example usage below the license.

Reply:
example usage should be in the printed message too, e.g. at the bottom.   that's where we normally put it.